### ISSUE TITLE
Latest Flatpak has broken functionality

### ISSUE THREAD

**Issue by starbock** [smplayer-dev/smplayer#901]
The latest Flatpak release has broken functionality.  In specific, Video | {Rotate, Flip Image, Mirror Image} do nothing when activated.

**To Reproduce**
1. Go to Video from the menu
2. Click on {Rotate, Flip Image, Mirror Image}
4. Instead of the video rotating, or flipping vertically / horizontally, the video remains unchanged

**Expected behavior**
Rotate, Flip Image, and Mirror Image work.

**Your Enviroment**
 - SMPlayer version:  23.12.0 (revision 10207)
 - OS: Linux Mint 21.3 Cinnamon
 - Multimedia engine you're using with SMPlayer (preferences -> general):
 [ ] MPlayer
 [ ] MPV
 [X] Other (it points to /app/bin/mpv, which is incorrect ... and I can't change it ... browsing to /usr/bin/mpv doesn't persist after I apply/save, neither does entering /usr/bin/mpv ... it always reverts back to /app/bin/mpv, and selecting MPV from the dropdown list also reverts back to Other /app/bin/mpv after apply/save, too)

**Logs**
If I run interactively I don't see any errors thrown when I invoke the Video | {Rotate, Flip Image, Mirror Image} functions.

**Additional context**
This functionality was working fine prior to the Flatpak 12/28/2023 update.  I assumed that I had an issue with my keyboard mappings, but after I got around to doing some troubleshooting I realized that the menu navigation doesn't work either.

**Comment by smplayer-dev** [comment]
The /app/bin path is correct, it's where the binaries are installed in flatpak packages.
I can't reproduce this problem with rotate, flip and mirror in Ubuntu 22.04.

**Comment by starbock** [comment]
> The /app/bin path is correct, it's where the binaries are installed in flatpak packages. I can't reproduce this problem with rotate, flip and mirror in Ubuntu 22.04.

Thanks for the /app/bin context and for checking!  I uninstalled (including Flatsweep after) and re-installed ... same thing (and my settings were preserved, which was a bit surprising).

I tried running the Flatpak directly (flatpak run info.smplayer.SMPlayer instead of through file association autorun), and, yes, Video | {Rotate, Flip Image, Mirror Image} work.  However, the UI looks completely different (looks like settings revert to default when run directly like this).  Odd that this happened after the Flatpak was upgraded - it'd always worked w/o any issues.

So, I'll work on it more to figure out what setting where is interfering.  Thanks!
